src/ghostfat.c

```diff
@@ -136,20 +136,26 @@ const char indexFile[] =
     "</body>"
     "</html>\n";
 
-static struct TextFile const info[] = {
-    {.name = "INFO_UF2TXT", .content = infoUf2File},
-    {.name = "INDEX   HTM", .content = indexFile},
-
+#if TINYUF2_FAVICON
+#include "favicon.h"
+const char autorunFile[] = "[Autorun]\r\nIcon=FAVICON.ICO\r\n";
+#endif
+
+static TextFile_t const info[] = {
+    {.name = "INFO_UF2TXT", .content = infoUf2File, .size = sizeof(infoUf2File) - 1},
+    {.name = "INDEX   HTM", .content = indexFile  , .size = sizeof(indexFile  ) - 1},
+#if TINYUF2_FAVICON
+    {.name = "AUTORUN INF", .content = autorunFile, .size = sizeof(autorunFile) - 1},
+    {.name = "FAVICON ICO", .content = (char const *) favicon_data, .size = favicon_len},
+#endif
     // current.uf2 must be the last element and its content must be NULL
-    {.name = "CURRENT UF2", .content = NULL},
+    {.name = "CURRENT UF2", .content = NULL       , .size = 0},
```

Comment:
I'm recommending later that the `size` field `CURRENT.UF2` should be filled in.  See later for why

Reply:
I am making this change in a branch, so you can review easily.

Reply:
Yeah, I agree, that makes it more consistent